Zend/zend_execute.c

```diff
@@ -1232,6 +1232,22 @@ ZEND_API ZEND_COLD void zend_verify_return_error(const zend_function *zf, zval *
 	zend_string_release(need_msg);
 }
 
+ZEND_API ZEND_COLD void zend_verify_noreturn_error(const zend_function *zf)
+{
+	const zend_arg_info *arg_info = &zf->common.arg_info[-1];
+	const char *fname, *fsep, *fclass;
+	zend_string *need_msg;
+	const char *given_msg;
+
+	zend_verify_type_error_common(
+		zf, arg_info, NULL, &fname, &fsep, &fclass, &need_msg, &given_msg);
```

Comment:
what is the reason for this check?

Reply:
As that's the only part you need, you can use `get_function_or_method_name(zf)` to fetch the function/method prefix for the error.